### DISCUSSION TITLE
Dewpoint and Relative Humidity missing from Forecast API

### DISCUSSION THREAD

**Discussion by ChrisAtQuest** [weather-gov/api#752] [General]
I have had a feed working for quite some time and starting sometime last week, the Dewpoint and Relative Humidity is no longer in the "periods"  feed.  According to the documentation example it should still be there.  As an example, here is one of the sites I pull:  [https://api.weather.gov/gridpoints/LWX/83,69/forecast] .  This gives me the following, which as of today is missing the dewpoint and relative humidity.   Is this a new change?

{
    "@context": [
        "https://geojson.org/geojson-ld/geojson-context.jsonld",
        {
            "@version": "1.1",
            "wx": "https://api.weather.gov/ontology#",
            "geo": "http://www.opengis.net/ont/geosparql#",
            "unit": "http://codes.wmo.int/common/unit/",
            "@vocab": "https://api.weather.gov/ontology#"
        }
    ],
    "type": "Feature",
    "geometry": {
        "type": "Polygon",
        "coordinates": [
            [
                [
                    -77.439337300000005,
                    38.897386599999997
                ],
                [
                    -77.44301320000001,
                    38.8754177
                ],
                [
                    -77.414820600000013,
                    38.872557
                ],
                [
                    -77.411138900000012,
                    38.894525600000001
                ],
                [
                    -77.439337300000005,
                    38.897386599999997
                ]
            ]
        ]
    },
    "properties": {
        "units": "us",
        "forecastGenerator": "BaselineForecastGenerator",
        "generatedAt": "2024-08-12T17:24:37+00:00",
        "updateTime": "2024-08-12T14:30:23+00:00",
        "validTimes": "2024-08-12T08:00:00+00:00/P7DT17H",
        "elevation": {
            "unitCode": "wmoUnit:m",
            "value": 91.135199999999998
        },
        "periods": [
            {
                "number": 1,
                "name": "This Afternoon",
                "startTime": "2024-08-12T13:00:00-04:00",
                "endTime": "2024-08-12T18:00:00-04:00",
                "isDaytime": true,
                "temperature": 86,
                "temperatureUnit": "F",
                "temperatureTrend": "",
                "probabilityOfPrecipitation": {
                    "unitCode": "wmoUnit:percent",
                    "value": null
                },
                "windSpeed": "7 mph",
                "windDirection": "NW",
                "icon": "https://api.weather.gov/icons/land/day/sct?size=medium",
                "shortForecast": "Mostly Sunny",
                "detailedForecast": "Mostly sunny, with a high near 86. Northwest wind around 7 mph."
            },
            {
                "number": 2,
                "name": "Tonight",
                "startTime": "2024-08-12T18:00:00-04:00",
                "endTime": "2024-08-13T06:00:00-04:00",
                "isDaytime": false,
                "temperature": 63,
                "temperatureUnit": "F",
                "temperatureTrend": "",
                "probabilityOfPrecipitation": {
                    "unitCode": "wmoUnit:percent",
                    "value": null
                },
                "windSpeed": "2 to 7 mph",
                "windDirection": "NW",
                "icon": "https://api.weather.gov/icons/land/night/sct?size=medium",
                "shortForecast": "Partly Cloudy",
                "detailedForecast": "Partly cloudy, with a low around 63. Northwest wind 2 to 7 mph."
            },

**Comment by chaunceygardiner** [comment]
The change was announced in May.  They still exist one the hourly forecast periods.

NOUS41 KWBC 221600
PNSWSH
Service Change Notice 24-55
National Weather Service Headquarters Silver Spring MD
1200 PM EDT Wed May 22 2024
TO: Subscribers:
-NOAA Weather Wire Service
-Emergency Managers Weather Information Network
-NOAAPort
Other NWS Partners, Users and NWS Employees
FROM: Ben Kyger
Director, NCEP Central Operations (NCO)
SUBJECT: Upgrade NWS API Website Forecast and Alerts Endpoints,
Effective June 20th, 2024
On or about June 20th, 2024 at 1400 Universal Coordinated Time (UTC), the
National Weather Service API Application Programming Interface (Website)
will be upgraded to version 1.13. With this upgrade comes the following
user-impactful changes:

Removal of the following parameters from the
/gridpoint//,/forecast endpoint:
updated, Relative Humidity, and Dewpoint
Example: https://api.weather.gov/gridpoints/EAX/45,35/forecast
A) Users who are using the “updated” parameter in their parsing of the
data can use the “updateTime” field which shows the same time.
B) The Relative Humidity and Dewpoint values are being removed as they
are not consistent with the 12 hour periods as currently presented.
These parameters will still be available on the
/gridpoint//, and /gridpoint//,/forecast/hourly
endpoints.
Requests to the alerts endpoint with the active and zone parameters
will be redirected to the discrete endpoint.
Example:
https://api.weather.gov/alerts?active=true&zone=WYZ118
Will be redirected to:
https://api.weather.gov/alerts/active/zone/WYZ118
Additionally, users are advised that the active parameter on the /alerts
endpoint is deprecated and is subject to removal in a future update of the
API. The /alerts/active endpoint should be used as the replacement.
Resolution of WFO Anchorage (AFC) errors on the points and gridpoints
endpoints.
Examples of endpoints that are being resolved:
https://api.weather.gov/points/61.2175,-149.8583
This will now return values for forecast, forecastHourly,
forecastGridData, and observations.
https://api.weather.gov/gridpoints/AER/126,236/forecast
and
https://api.weather.gov/gridpoints/AER/126,236/forecast/hourly
also will return data.
For operational issues on the API Website, please contact:
NCEP Central Operations
Tech Control
[nco.ops@noaa.gov](mailto:nco.ops@noaa.gov)
For technical questions regarding this notification, please contact:
Anne Myckow
Software Development Branch
NWS Office of Dissemination
[sdb.support@noaa.gov](mailto:sdb.support@noaa.gov)
National Service Change Notices are online at:
https://www.weather.gov/notification
NNNN

**Comment by AdamPloof** [comment]
Just mentioning that the documentation for the [/gridpoints/{wfo}/{x},{y}/forecast](https://www.weather.gov/documentation/services-web-api#/default/gridpoint_forecast) endpoint still shows the `relativeHumidity` and `dewpoint` parameters in the example response.

**Comment by sullynole** [comment]
> Just mentioning that the documentation for the [/gridpoints/{wfo}/{x},{y}/forecast](https://www.weather.gov/documentation/services-web-api#/default/gridpoint_forecast) endpoint still shows the `relativeHumidity` and `dewpoint` parameters in the example response.

Thank you.  This has been documented for removal.
